pkg/persistence/postgres/event.go

```diff
@@ -101,8 +101,14 @@ func (c *Event) EventDataToDB(event map[string]interface{}) error {
 		case "address":
 			eventPayload[eventFieldName] = eventField.(common.Address).Hex()
 		case "uint256":
-			// NOTE(IS): converting all *big.Int to int64. assuming for now that numbers will fall into int64 range.
-			eventPayload[eventFieldName] = eventField.(*big.Int).Int64()
+			i := eventField.(*big.Int)
```

Comment:
Do we have a test for this field?  What does this get saved as in the DB?

Reply:
Also, should we consider just storing this as a string to keep it consistent rather than potentially manage two different types?   Would there be a scenario where we need it to be a numeric value in some query?

Reply:
Oh, I thought we could just store values that overflowed int64 as float64. I'm fine with either, I can just convert that string value back to a big.int. Will change it then

Reply:
Yeah there's no scenario where we'd need it as a numeric value. The only thing is that this will convert all uint256 values to string unless we case handle for "deposit", or other fields we find

Reply:
ok, let's see if we can keep it a value.  Yeah, as you suggested, maybe we just treat this field as a float64 rather than determining if it is an int vs a float.  In this case, store all the values as floats.   I don't know if you did this already, but test to see if the value stores down to the DB properly.

Reply:
Cool, changed to convert all uint256 types to float64 for saving to DB which makes more sense bc postgres was storing those jsonb numbers as floats anyway